pkg/template/master_template.go

```diff
@@ -194,7 +198,7 @@ systemd:
           --name $NAME \
           $IMAGE \
           etcd \
-          --name {{ .Etcd.NodeName }} \
+          --name ${ETCD_NAME} \
```

Comment:
I would suggest to test his as I fear that systemd itself won't fill the ${} values as you might need to run it whoel in bash wrapper `/bin/bash -c "docker run ...."`.

Or at least I know this was a problem somewhere. (maybe just in defining pod args)

Reply:
I tested this in the last days, it should work.

Reply:
Cool